jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/timstorage/TravelerDataFrame.java

```diff
@@ -1,169 +1,168 @@
+/**
+ * Traveler data frame.
+ */
+@JsonPropertyOrder({"doNotUse1", "frameType", "msgId", "startYear", "startTime", "durationTime",
+    "priority",
+    "doNotUse2", "regions", "doNotUse3", "doNotUse4", "tcontent", "url"})
+@EqualsAndHashCode(callSuper = false)
 public class TravelerDataFrame extends Asn1Object {
-   private static final long serialVersionUID = 1L;
-
-   private String notUsed;
-
-   private FrameType frameType;
-
-   private MsgId msgId;
-
-   private String startYear;
-
-   private String startTime;
-
-   private String durationTime;
-
-   private String priority;
-
-   private String notUsed1;
-
-   private Regions regions;
-
-   private String notUsed2;
-
-   private String notUsed3;
-
-   @JsonProperty("tcontent")
-   private Content tcontent;
-
-   private String url;
-
-   public String getNotUsed1() {
-      return notUsed1;
-   }
-
-   public void setNotUsed1(String sspLocationRights) {
-      this.notUsed1 = sspLocationRights;
-   }
-
-
-   public Regions getRegions() {
-     return regions;
-   }
-
-   public void setRegions(Regions regions) {
-     this.regions = regions;
-   }
-
-   public String getUrl() {
-      return url;
-   }
-
-   public void setUrl(String url) {
-      this.url = url;
-   }
-
-   public FrameType getFrameType() {
-      return frameType;
-   }
-
-   public void setFrameType(FrameType frameType) {
-      this.frameType = frameType;
-   }
-
-   public String getDurationTime() {
-      return durationTime;
-   }
-
-   public void setDurationTime(String duratonTime) {
-      this.durationTime = duratonTime;
-   }
-
-   @JsonIgnore
-   public Content getContent() {
-      return tcontent;
-   }
-
-   public Content getTcontent() {
-     return tcontent;
-   }
-
-   public void setTcontent(Content tcontent) {
-     this.tcontent = tcontent;
-   }
-
-   public void setContent(Content content) {
-      this.tcontent = content;
-   }
-
-   public String getStartTime() {
-      return startTime;
-   }
-
-   public void setStartTime(String startTime) {
-      this.startTime = startTime;
-   }
-
-   public String getStartYear() {
-      return startYear;
-   }
-
-   public void setStartYear(String startYear) {
-      this.startYear = startYear;
-   }
-
-   public String getPriority() {
-      return priority;
-   }
-
-   public void setPriority(String priority) {
-      this.priority = priority;
-   }
-
-   public MsgId getMsgId() {
-      return msgId;
-   }
-
-   public void setMsgId(MsgId msgId) {
-      this.msgId = msgId;
-   }
-
-   public String getNotUsed3() {
-      return notUsed3;
-   }
-
-   public void setNotUsed3(String sspMsgRights2) {
-      this.notUsed3 = sspMsgRights2;
-   }
+  private static final long serialVersionUID = 1L;
+  @JsonAlias({"sspTimRights", "notUsed"})
+  private String doNotUse1;
+  private FrameType frameType;
+  private MsgId msgId;
+  private String startYear;
+  private String startTime;
+  @JsonAlias("duratonTime")
```

Comment:
Since this was renamed in 2020 migration, should we leave it as correct spelling?

Reply:
Ah I see, this is allowing backwards compatibility?

Reply:
The `@JsonAlias` annotation allows Jackson to accept both `durationTime` and `duratonTime` when deserializing JSON into this model.